pkg/sources/README.md

```diff
@@ -0,0 +1,14 @@
+# pkg/sources
+
+This package will build a docker image that will store the collected_sources.tar.gz which is needed for SBOM comparison. The docker image will be pushed to `eve-sources` repo under lfedfge on dockerhub.
+
+Collected_sources.tar.gz includes:
+
+* alpine, kernel and golang packages.
+* A manifest file collected_sources_manifest.csv that holds the metadata for all the packages.
+
+## Usage
+
+```bash
+make -e V=1 HV=${{ matrix.hv }} ZARCH=${{ matrix.arch }} LINUXKIT_PKG_TARGET=push publish_sources
```

Comment:
Can you add something about how this does not run locally, but rather is copied to `SOURCES_DIR` from the `Makefile`, e.g. `dist/amd64/0.0.0-master-..../sources/` and run there, so no one gets confused?

Reply:
Will add more info.

Reply:
added more around the workflow and process